pkg/aggregator/aggregator.go

```diff
@@ -121,6 +126,8 @@ var (
 	// Hold series to be added to aggregated series on each flush
 	recurrentSeries     metrics.Series
 	recurrentSeriesLock sync.Mutex
+	// ensures event platform errors are logged at most once per flush
+	aggregatorEventPlatformErrorLogged bool
```

Comment:
This should be a local variable to the `run` function since it's only used there and we might instantiate multiple aggregator one day.